rwengine/src/engine/GameState.cpp

```diff
@@ -20,7 +20,7 @@ BasicState::BasicState()
     , timeScale2{0}
     , lastWeather{0}
     , nextWeather{0}
-    , forcedWeather{0}
+    , forcedWeather{UINT16_MAX}
```

Comment:
nit: Personally I don't like initializing to a constant like this

Reply:
You prefer 65535?

Reply:
No, that's equally bad :P why not just 0 like it used to be? I did not remember your comments about weather, but can't this even be a boolean?

Reply:
I changed this to UINT16_MAX because based on my research on save games; the forced weather value was 0xFFFF when weather was not forced with ie. weather cheat. If weather cheat was used then the value was for example 0x0003 when forced to foggy.

Reply:
then use... 0xFF or add an enum with a value like: `WEATHER_CYCLING = 0xFF`